gettsim/functions_loader.py

```diff
@@ -23,12 +34,12 @@ def convert_paths_to_import_strings(paths):
     return import_strings
 
 
-def load_functions(sources, allow_imported_members=False):
+def _load_functions(sources, allow_imported_members=False):
     """Load functions.
 
     Parameters
     ----------
-    sources : str, pathlib.Path, function, module, dictionary of functions
+    sources : str, pathlib.Path, function, module, imports statements
```

Comment:
```suggestion
    sources : str, pathlib.Path, function, module
```

See above, did not check the code below.